tests/manager/feed/test_annotators.py

```diff
@@ -455,6 +483,16 @@ def test_annotate_work_entry(self, db: DatabaseTransactionFixture):
         assert data.language == None
         assert data.updated == FeedEntryType(text=strftime(now))
 
+        # other links
+        other_links = data.other_links
+        assert other_links[0].type == MediaTypes.EPUB_MEDIA_TYPE
+        assert other_links[0].href.startswith("http://epub-a")
+        assert other_links[1].type == MediaTypes.EPUB_MEDIA_TYPE
+        assert other_links[1].href.startswith("http://epub-b")
```

Comment:
The `href`s here have type `str | None`, so we need to handle the None case. A couple ways to do that:
```python
assert other_links[0].href is not None
assert other_links[0].href.startswith("http://epub-a")
```
or 
```python
assert other_links[0].href and other_links[0].href.startswith("http://epub-a")
```